lib/hammer_cli_foreman/keycloak.rb

```diff
@@ -0,0 +1,52 @@
+require 'net/http'
+require 'uri'
+module HammerCLIForeman
+
+  class Keycloak
+
+    def initialize(url, realm, client_id)
+      @url = url
+      @realm = realm
+      @client_id = client_id
+    end
+
+    def get_token(username, password)
+      uri = URI.parse(token_url)
```

Comment:
Could we add some logging in here saying what is the url we use to get token and what contained the response.